core/store.go

```diff
@@ -328,3 +329,18 @@ func delByPtr(ptr unsafe.Pointer) bool {
 	}
 	return false
 }
+
+func RandomKey() string {
```

Comment:
Do we know if accessing a key randomly effects the expiry of the key?

Reply:
@soumya-codes yes RANDOMKEY should not return expired keys (with maxtries of 100, if exceeds max try it may return the expired key)